internal/scaffold/crd.go

```diff
@@ -15,6 +15,7 @@
 package scaffold
 
 import (
+	goerrors "errors"
```

Comment:
why create an alias ? Could not it jus be : 
```suggestion
 	"errors"
```

Reply:
We're already using `github.com/pkg/errors` in this file, so one of them needs an alias. Rather than changing all of the existing usages, I thought it would be simpler and easier to review to use an alias here.